coding-concepts.js

```diff
@@ -9,51 +9,51 @@
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // console.log(colors.push("indigo"))
 
-// a) Your answer:
+// a) Your answer:l
 // b) Verify and explain:
```

Comment:
Make sure to check the answer and explain!  Even if you did not get it right, this process helps to gain a deeper understanding.